client/boot/index.js

```diff
@@ -161,6 +161,20 @@ function boot() {
 	createReduxStoreFromPersistedInitialState( reduxStoreReady );
 }
 
+export function addLayoutToContext( context, next ) {
```

Comment:
This has some duplication with `reduxStoreReady` below. Probably doesn't belong in this module, since it will be used on every page change, not just boot.

It adds a load of mess to `context`.